.ci/prepare-settings.sh

```diff
@@ -20,7 +20,7 @@ replace SONATYPE_PWD "$SONATYPE_PWD"
 replace GPG_PASSPHRASE "$GPG_PASSPHRASE"
 replace GPG_KEY "$GPG_KEY"
 
-
+mkdir -p ~/.m2
```

Comment:
We assume that `~/.m2` directory exists but that is not the case on runners:

https://github.com/stoyanK7/checkstyle/actions/runs/4195984802/jobs/7276471446#step:5:10
```
cp: cannot create regular file '/home/runner/.m2/settings.xml': No such file or directory
```